README.md

```diff
@@ -79,38 +79,36 @@ sudo ./install-dependencies.sh && CC=clang CXX=clang++ ./build.sh
 
 For quicker dev setup, we provide a [docker image](tools/docker/README.md) with the toolchain installed.
 
-# Beta builds
+# Release candidate builds
 
 For those of us who like to live on the edge!
+We cut RCs when we get close to a new release, and publish these in case you want to test a particular feature.
 
-We can cut a release at any point from the `/dev` branch if you want to test a particular feature.
-Simply let us know you would like to test a feature from dev and we're happy to cut a beta release.
 
+## RC releases on Debian/Ubuntu
 
-## Beta releases on Debian/Ubuntu
-
-```
+```bash
 curl -1sLf \
-  'https://packages.vectorized.io/HxYRCzL4xbbaEtPi/redpanda-beta/setup.deb.sh' \
+  'https://packages.vectorized.io/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.deb.sh' \
```

Comment:
I don’t get why change the upstream repo link seems like no need.  What am I missing.

Reply:
i initially thought of renaming the repo but that'll break any exiting config. should we do that instead?